# Accelerometer [m/s^2]

geometry_msgs/Vector3 accel

# Gyroscope [rad/s]

geometry_msgs/Vector3 gyro

# Internal Temperature [Â°C]
float64 temp

# delta velocity output [m/s]

geometry_msgs/Vector3 delta_vel

# delta angle output [degrees]

geometry_msgs/Vector3 delta_angle

